### DISCUSSION TITLE
Formatting the DataGrid popup edit form

### DISCUSSION THREAD

**Discussion by GilShalit** [Megabit/Blazorise#2478] [Q&A]
Hi,

Is it possible to modify the popup Edit form in the DataGrid?  Would be nice to be able to increase size, have 3 columns of items, etc.

**Comment by stsrki** [comment]
Currently, it's only possible to change column size on edit column, by using `PopupFieldColumnSize` eg

```cshtml
<DataGridSelectColumn TItem="UserViewModel" Field="@nameof(UserViewModel.RoleNames)" Caption="Roles" Editable PopupFieldColumnSize="ColumnSize.Is6">
```

Other than that, the only other way is to use EditTemplate for customization.
